README.md

```diff
@@ -127,40 +127,40 @@ If you want to build these quickstarts from scratch, please follow the articles
+| [Python Console app](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/python/console/)                                                              | Windows, Linux, macOS  | Demonstrates speech recognition, intent recognition, and translation |
+| [Speech-to-text WPF sample for .NET Framework on Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/dotnet-windows/speechtotext-wpf/) | Windows  | Demonstrates speech recognition |
+| [Translation WPF sample for .NET Framework on Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/dotnet-windows/translation-wpf/)     | Windows  | Demonstrates translation |
+| [Speech-to-text UWP sample](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/uwp/speechtotext-uwp/)                                          | Windows  | Demonstrates speech recognition |
+| [Text-to-speech UWP sample](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/uwp/texttospeech-uwp/)                                          | Windows  | Demonstrates speech synthesis |
+| [Speech recognition sample for Android](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/java/android/sdkdemo/)                                     | Android  | Demonstrates speech and intent recognition |
+| [Speech recognition and translation sample for the browser, using JavaScript](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/js/browser/)         | Web      | Demonstrates speech recognition, intent recognition, and translation |
+| [Speech recognition and translation sample using JavaScript and Node.js](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/js/node/)                 | Node.js  | Demonstrates speech recognition, intent recognition, and translation |
+| [Speech recognition sample for iOS using a connection object](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/objective-c/ios/README.md#connection-object-sample)  | iOS      | Demonstrates speech recognition |
+| [Extended speech recognition sample for iOS](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/objective-c/ios/README.md#example-app-with-various-samples)           | iOS      | Demonstrates speech recognition using streams etc.|
+| [C# UWP DialogServiceConnector sample for Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/virtual-assistant/csharp-uwp/) | Windows | Demonstrates speech recognition through the DialogServiceConnector and receiving activity responses. |
```

Comment:
>https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/virtual-assistant/csharp-uwp/ [](start = 53, length = 112)

need to double check this one too

Reply:
samples\csharp\uwp\virtualassistant-uwp

?

---
In reply to: [310005274](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/327#discussion_r310005274) [](ancestors = 310005274)

Reply:
https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/uwp/virtualassistant-uwp/

Thanks!!